### DISCUSSION TITLE
Daphne ping pong

### DISCUSSION THREAD

**Discussion by Producks** [django/daphne#496] [Q&A] [answered]
Hi, sorry for the noobie question.
I've been wondering if there a way to check if the ping pong for daphne is happening? I tried checking in chrome under the Network tab, then I selected my socket and under messages it never shows any sign on PING or PONG.

**Comment by carltongibson** [comment, marked as answer]
Hi. 

I assuming you're using something like [the `PingConsumer` from the Channels docs](https://channels.readthedocs.io/en/stable/introduction.html)... 

```python
class PingConsumer(AsyncConsumer):
    async def websocket_connect(self, message):
        await self.send({
            "type": "websocket.accept",
        })

    async def websocket_receive(self, message):
        await asyncio.sleep(1)
        await self.send({
            "type": "websocket.send",
            "text": "pong",
        })
```

If so, you can add some logging (or print output even) to `websocket_receive` to show the events in action.

**Reply by Producks** [reply]
Thanks a lot.
